.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release Charts
```

Comment:
I think it will be ok to use GitHub actions to automate the build and deploy of the helm chart at release time. We will have to see what the other project maintainers think about this.

Reply:
Please see my comments on releasing above.